RecoLocalTracker/Phase2TrackerRecHits/python/Phase2StripCPEESProducer_cfi.py

```diff
@@ -2,5 +2,8 @@
 
 phase2StripCPEESProducer = cms.ESProducer("Phase2StripCPEESProducer",
   ComponentType = cms.string('Phase2StripCPE'),
-  parameters    = cms.PSet()
+  parameters    = cms.PSet(
+    LorentzAngle_DB = cms.bool(False),			
+    TanLorentzAnglePerTesla = cms.double(0.07)
```

Comment:
The pixel LA is 0.106 [https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py#L43], while long pixel [https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py#L79], strip in PS [https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py#L115], and strips in 2S [https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py#L151] are all 0.07.